src/main/java/vinegar/Parser.java

```diff
@@ -0,0 +1,31 @@
+package vinegar;
+
+import vinegar.command.*;
+
+public class Parser {
+    public static Command parse(String fullCommand) throws VinegarException {
+        String[] inputParts = fullCommand.split(" ", 2);
+        String instruction = inputParts[0].toLowerCase();
+
+        switch (instruction) {
+            case "bye":
+                return new ExitCommand();
+            case "list":
+                return new ListCommand();
+            case "mark":
+                return new MarkCommand(inputParts);
+            case "unmark":
+                return new UnmarkCommand(inputParts);
+            case "todo":
+                return new AddTodoCommand(inputParts);
+            case "deadline":
+                return new AddDeadlineCommand(inputParts);
+            case "event":
+                return new AddEventCommand(inputParts);
+            case "delete":
+                return new DeleteCommand(inputParts);
+            default:
```

Comment:
Should the `case` and `default` keywords for the a `switch` statement be on same indentation level? Did you [configure](https://se-education.org/guides/tutorials/intellijCodeStyle.html#tweak-switch-case-style) your IDE correctly?